pkg/services/alerting/eval_handler_test.go

```diff
@@ -153,7 +153,7 @@ func TestAlertingEvaluationHandler(t *testing.T) {
 			So(context.ConditionEvals, ShouldEqual, "[[false OR false] OR false] = false")
 		})
 
-		Convey("Should retuasdfrn no data if one condition has nodata", func() {
+		Convey("Should return no data if one condition has nodata using OR operator", func() {
```

Comment:
Because the same precise convey string is used in the next convey call, so I needed to do something, otherwise the CI failed.

I don't care what the words were, but I needed something.

The code only accidentally worked due to that keyboard inserted damage.

Reply:
https://github.com/grafana/grafana/commit/bf999a4f826aab98c3ecfea94bbd16cfb4d1abc7#diff-34cb3b27b2b7c3d3f53d31e812bbd169R158-R171

@bergquist